tensorflow/python/ops/data_flow_ops.py

```diff
@@ -2099,6 +2100,10 @@ def __init__(self,
       file_pattern: File path to the dataset, possibly containing wildcards.
         All matching files will be iterated over each epoch.
       batch_size: How many records to return at a time.
+      minibatches: None by default, otherwise specifies
```

Comment:
There could still be a larger concept of batch size assuming there are multiple machines involved, which is why I think the least ambiguous option is to use batch size to describe either the largest or smallest grouping.

Perhaps we can slip the issue by avoiding ML terminology all together, since really we're just trying to ask for N groups of size K.

records_per_group
num_groups